tests/unit/plugins/cache/test_redis.py

```diff
@@ -25,8 +25,21 @@
 
 from ansible.plugins.loader import cache_loader
 from ansible_collections.community.general.plugins.cache.redis import CacheModule as RedisCache
+from ansible import constants as C
+from ansible.release import __version__ as ansible_version
```

Comment:
How about ordering the imports? :)
```suggestion
from ansible import constants as C
from ansible.plugins.loader import cache_loader
from ansible.release import __version__ as ansible_version
from ansible_collections.community.general.plugins.cache.redis import CacheModule as RedisCache
```